test/ci/buildkite/run_pipeline.sh

```diff
@@ -7,12 +7,12 @@ echo "BUILDKITE_REPO: $BUILDKITE_REPO"
 echo "PIPELINE OS: $PIPELINE_QUEUE"
 
 if [[ $PIPELINE_QUEUE = 'cpu' ]]; then
-   TF_PY_WHEEL=tensorflow-1.15.2-cp35-cp35m-linux_x86_64.whl
+   TF_PY_WHEEL=tensorflow-2.0.0-cp36-cp36m-linux_x86_64.whl
    # For the time being - hardcode the file
    # Eventually we will replace the queue and other variables during the pipeline creation
    STEPS_FILE=ngtf-cpu_ubuntu.yaml
 elif [[ $PIPELINE_QUEUE = 'cpu-centos' ]]; then
-   TF_PY_WHEEL=tensorflow-1.15.2-cp36-cp36m-linux_x86_64.whl
+   TF_PY_WHEEL=tensorflow-2.0.0-cp36-cp36m-linux_x86_64.whl
```

Comment:
high level, seems like we are always installing 2.0. is this right? Not sure what this file does.

Reply:
I am not very sure of this file's purpose either but when you look at the CI pipelines build step, the version printed in the end is 1.15